components/core/src/clp_s/RangeIndexWriter.cpp

```
@@ -0,0 +1,82 @@
+#include <utility>
+#include <vector>
+
+#include <nlohmann/json.hpp>
+#include <nlohmann/json_fwd.hpp>
+
+#include "ErrorCode.hpp"
+#include "SingleFileArchiveDefs.hpp"
+#include "ZstdCompressor.hpp"
+
+namespace clp_s {
+auto RangeIndexWriter::open_range(size_t start_index) -> ErrorCode {
```

Cppcheck (2.10-2)

[style] The function 'open_range' is never used.

(unusedFunction)